Sprint-3/revise/implement/is-prime.test.js

```diff
@@ -1,3 +1,36 @@
 // Given a positive integer num,
 // When the isPrime function is called with num as input,
+
+//const { default: test } = require("node:test");
+//const { hasUncaughtExceptionCaptureCallback } = require("process");
+
 // Then it should check if the num is prime
+function isPrime(num) {
+  if (num <= 1) return "False";
+  if (num === 2) return "True";
+  for (let i = 2; i <= Math.sqrt(num); i++) {
```

Comment:
You can possibly improve the performance of the code in the following manners:
- Check if num is 2, and check only odd numbers >= 3 in the loop
- Avoid calling `Math.sqrt(num)` repeatedly by assigning the value of `Math.sqrt(num)` to a variable once, and then refer to the variable in the condition of the loop.
  - Note: The condition is checked at the start of every iteration.